incubator/fluentd-cloudwatch/values.yaml

```diff
@@ -0,0 +1,26 @@
+image: 18fgsa/fluentd-cloudwatch
+imageTag: 0.1.0
+namespace: kube-system
```

Comment:
Is the `kube-system` default a hard requirement? Generally we prefer the namespace be managed by `helm` using the `--namespace` flag.